quiz.rb

```diff
@@ -2,19 +2,66 @@
 # Create a class called Dog. It should have, as attributes, name, breed, and age.
 # Make all of those attributes read-only.
 
+class Dog
+  attr_reader :name, :breed, :age
+  def initialize(name, breed, age)
+    @name = name
+    @breed = breed
+    @age = age
+  end
+end
+
+
 # Question 2
 # Create a new Dog object.
 
+fluffy = Dog.new("fluffy", "lab", 2)
+
 # Question 3
 # Create a Person class, with attributes name, age, and location.
 # It should set all three attributes when you create a new Person.
 # Name and location should be writeable, but age should be read-only.
 
+class Person
+  def initialize(age)
```

Comment:
We want to set all three attributes when we create a new person, so this means that our `initialize` method should take three arguments and use those as values for `name`, `age`, and `location`; it should look almost the same as the `initialize` method from question 1.